code/modules/mob/living/carbon/human/species/drask.dm

```diff
@@ -45,7 +45,7 @@
 	heat_level_1 = 310 //Default 370 - Higher is better
 	heat_level_2 = 340 //Default 400
 	heat_level_3 = 400 //Default 460
-	heatmod = 3 // 4.5 * more damage from body temp
+	heatmod = 2 // 3.5 * more damage from body temp
```

Comment:
```suggestion
	heatmod = 2 // 3 * more damage from body temp
```
2 * 1.5 = 3
👍

Reply:
I swear I know maths, heh! Thank you! :grinning: